aip/general/0122.md

```diff
@@ -296,9 +296,8 @@ The field **should not** be of type `message` using the `message` that
 implements the resource, __except__ for one of following conditions:
 
 - The API is internal-only, has tight lifecycle relationships, and has a
-permission model that enables inherited access to embedded resources.
-- The resource is a revision as defined in AIP-162, where an embedded resource
-  is required.
+  permission model that enables inherited access to embedded resources.
+- The embedded resource has a [field behavior][] of `OUTPUT_ONLY`.
```

Comment:
I don't think this change should be made here. Only exempt the Revisions use case and we can consider opening to broader usage separately. This increases the scope of this PR significantly.

Reply:
ack, reverted to just mention revisions.